Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION}
+
+ENV ENV="Local"
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY . .
+
+RUN python3 manage.py migrate
```

Comment:
Running `manage.py migrate` during the build process can lead to issues if the database is not available or properly configured at build time. Consider running migrations as part of the container startup process instead.

Reply:
It is work fine and my mentor show me exectly this method for COPY. It is almost correct. Forget it